templatetags/htk_tags.py

```diff
@@ -302,6 +302,23 @@ def get_request_duration():
     return duration
 
 
+@register.simple_tag()
+def localize(key):
+    '''TODO: handle this based on `by_language`
+    for any language
+    '''
+    from htk.apps.i18n.utils.data import retrieve_all_strings
+
+    language_code = 'en-US'
+    localized_strings = retrieve_all_strings()
```

Comment:
Don't retrieve all strings just to extract 1; that's terribly inefficient.

Use the DB to look up just one record.

Reply:
Also, don't do the DB lookup directly here. Provide a utility function, and call that utility function here.